### DISCUSSION TITLE
Cloudflare DNS Challenge

### DISCUSSION THREAD

**Discussion by bearded-papa** [tobychui/zoraxy#145] [Q&A] [answered]
I am using Cloudflare with my current NGPM setup and had to use DNS challenge. How do I set this up in zoraxy as getting the certificates fails everytime

**Comment by tobychui** [comment, marked as answer]
@bearded-papa We are working on DNS validation for ACME in #144. I guess it will take another week to complete testing and be ready in the next Zoraxy release. 

Btw, if your Nginx Proxy Manager (NPM) is working perfectly in your setup, you should keep using it for now as Zoraxy is still in intense development and some features might be missing.
